pkg/security/security_profile/activity_tree/process_node_snapshot.go

```diff
@@ -20,6 +20,9 @@ import (
 	"syscall"
 	"time"
 
+	// shirou/gopsutil uses different logic for getting the memory maps Path
```

Comment:
And there is a performance issue as well I believe (which was the reason we stayed with the legacy one)